File: 프로그래머스/0/120869. 외계어 사전/외계어 사전.cs
using System;

public class Solution {
    public int solution(string[] spell, string[] dic) {
        int answer = 2;
        foreach(string d in dic)
        {
            bool isin = true;
            foreach(string s in spell)
            {
                if(!d.Contains(s))
                {
                    isin = false;
                    break;
                }
            }
            if(isin)
            {
                answer = 1;
                break;
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/120869. 외계어 사전/외계어 사전.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

int solution(vector<string> spell, vector<string> dic) {
    int answer = 2;
    for(auto d : dic)
    {
        bool isin = true;
        for(auto s : spell)
        {
            int it = d.find(s);
            if(it == string::npos)
            {
                isin = false;
                break;
            }
        }
        if(isin)
        {
            answer = 1;
            break;
        }
    }
    return answer;
}